conversions.go

```diff
@@ -954,7 +932,7 @@ func convertChildInodeEntry(
 	convertAttributes(in.Child, &in.Attributes, &out.Attr)
 }
 
-func convertFileMode(unixMode uint32) os.FileMode {
+func ConvertFileMode(unixMode uint32) os.FileMode {
```

Comment:
Here and below, please document all exported functions. Perhaps something like:

// ConvertFileMode returns an os.FileMode with the Go mode and permission bits set according to the Linux mode and permission bits.